contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -9894,3 +9879,12 @@ CheckDeclareAtAtGlobalVarName(const std::string name, int lineNr)
 		throw PGErrorWrapperException(ERROR, ERRCODE_SYNTAX_ERROR, format_errmsg("Incorrect syntax near '%s'.", name.c_str()), lineNr, 0);
 	}
 }
+
+static antlr4::tree::TerminalNode *
+getTokenFromFunctionOption(TSqlParser::Function_optionContext* o) {
+    if (o->execute_as_clause())
```

Comment:
can there be multiple options available?

Reply:
Yes, there can be multiple function options. We process each option individually using `getTokenFromFunctionOption` within `removeTokenFromOptionList()` method.